liquid-staking/src/setup/vote.rs

```diff
@@ -0,0 +1,20 @@
+use crate::basics::errors::ERROR_VOTE_SC_NOT_SET;
+
+multiversx_sc::imports!();
+
+#[multiversx_sc::module]
+pub trait VoteModule {
+    #[only_owner]
+    #[endpoint]
+    fn set_vote_contract(&self, sc_address: ManagedAddress) {
+        self.vote_contract().set(sc_address);
+    }
```

Comment:
Maybe check it's actually a SC address and non-zero too. Example: https://github.com/multiversx/mx-sovereign-sc/blob/main/common/utils/src/lib.rs#L15-L20